distributed-exceptions/src/main/java/io/axoniq/distributedexceptions/api/IssuedEvt.java

```diff
@@ -0,0 +1,27 @@
+package io.axoniq.distributedexceptions.api;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * @author Stefan Andjelkovic
+ */
+public class IssuedEvt {
```

Comment:
Should we see it as a good practice and always add `@Revision` to our Events?

Reply:
Revision of `null` is also a valid revision, there's no real need to put the annotation on all the events.